### DISCUSSION TITLE
My App works perfectly in Debug but doesn't load well in Release

### DISCUSSION THREAD

**Discussion by Virock** [unoplatform/uno#15914] [Q&A]
Hi everyone. I just finished my first Uno Platform app. It loads perfectly well in Debug mode but not so well in Release.
Here is a screenshot of what it looks like in Debug mode.

![image](https://github.com/unoplatform/uno/assets/31706335/cffc1d7c-8172-4e7a-83b5-5bc4a699cce1)

'a' points to a Syncfusion combobox.
'b' points to 2 'chips'.

I am using a fluent theme but I added the Material theme to the top so I could use the Material's chip in my app.
Here is my AppResources.xaml file's contents:

```
<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
          xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">

  <ResourceDictionary.MergedDictionaries>
    <MaterialToolkitTheme xmlns="using:Uno.Toolkit.UI.Material" />
    <!-- Load WinUI resources -->
    <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
    <!-- Load Uno.UI.Toolkit resources -->
    <ToolkitResources xmlns="using:Uno.Toolkit.UI" />
  </ResourceDictionary.MergedDictionaries>
  <!-- Add resources here -->

</ResourceDictionary>

```

In Release mode, it looks like this:
![image](https://github.com/unoplatform/uno/assets/31706335/b7b6da3f-a41b-4910-9b51-897566409a67)

'a' - The Syncfusion combobox doesn't have it's contents anymore
'b' - The chips don't appear when I add images anymore.

Note that when I add images, the logs show that the images were added successfully.
I have also registered the Syncfusion on app start:
```
Syncfusion.Licensing.SyncfusionLicenseProvider.RegisterLicense("my_license");
```

What am I doing wrong?

**Comment by jeromelaban** [comment]
Thanks for the report. This is likely a linker related behavior and you'll need to exclude assemblies or annotate your types that may be using reflection.

It looks like you're running in mac catalyst, and will find additional information [here](https://learn.microsoft.com/en-us/dotnet/core/deploying/trimming/trimming-options?pivots=dotnet-8-0) and [here](https://platform.uno/docs/articles/external/uno.wasm.bootstrap/doc/linker-configuration.html).

**Reply by Virock** [reply]
I'm trying to deploy on Windows, not mac catalyst. Thank you. I'll read about how to handle reflections..

**Reply by Virock** [reply]
I have done everything in those links you sent to me but it's still not working. Can you please tell me exactly what I need to do to fix this?

**Reply by agneszitte** [reply]
Hi @Virock! I'm sorry to learn that you're still encountering issues. I'll reach out to @jeromelaban for further assistance and we will get back to you with additional guidance as soon as possible.

**Reply by jeromelaban** [reply]
@Virock this is something that is specific to WinUI. At this time, we do not have enough information to help you. If you're able to provide a repro, please let us know.

**Reply by Virock** [reply]
Hi there. Thank you for getting back to me. I can't create a repo because this is my company's code but I made it work by using Binding instead of x:Bind. For some reason, it works flawlessly when I do not use compiled Bindings.
Thank you for everything.
